lib/parse.js

```diff
@@ -70,16 +71,44 @@ function getRanges( map ) {
 }
 
 /**
+ * Zero out the file checksum field for checksum calculation
+ * @private
+ * @param {String} value - input
+ * @returns {String} maskedValue
+ */
+function maskChecksum( value ) {
+
+  var pattern = /BmapFileChecksum/.test( value ) ?
+    /(<BmapFileChecksum>\s*)([a-z0-9]+)(\s*<\/BmapFileChecksum>)/i :
+    /(<BmapFileSHA1>\s*)([a-z0-9]+)(\s*<\/BmapFileSHA1>)/i
```

Comment:
Just to be pedantic, I guess these regexes should actually be using `[a-f0-9]+` rather than `[a-z0-9]+` ;-)